dashboard.html

```
@@ -129,9 +129,9 @@
         <header>
             <h1>🤖 SCALPER BRAIN <span class="badge sim">SIMULATION MODE</span></h1>
             <div style="text-align: right; display: flex; flex-direction: column; gap: 5px; align-items: flex-end;">
-                <span class="status-badge active" style="background-color: #00ff00; color: #000000; font-family: monospace; font-weight: bold;">[ACTIVE] Crypto Markets</span>
-                <span class="status-badge hold" style="background-color: #555555; color: #ffffff; font-family: monospace; font-weight: bold;">[CLOSED <= 4H] None</span>
-                <span class="status-badge active" style="background-color: #ff9900; color: #000000; font-family: monospace; font-weight: bold;">[UPCOMING] None</span>
+                <span class="status-badge active" style="background-color: #00ff00; color: #000000; font-family: monospace; font-weight: bold;">[ACTIVE] Wellington FX | Sydney FX | US After-Hours | CME Futures | Crypto Markets</span>
+                <span class="status-badge hold" style="background-color: #555555; color: #ffffff; font-family: monospace; font-weight: bold;">[CLOSED <= 4H] New York FX | São Paulo B3 | Mexican BMV | US NYSE/NASDAQ | ICE Brent</span>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)